src/parse-stream.js

```diff
@@ -103,336 +104,342 @@ export default class ParseStream extends Stream {
       return;
     }
 
-    for (let i = 0; i < this.customParsers.length; i++) {
-      if (this.customParsers[i].call(this, line)) {
-        return;
+    // map tags
+    const newLines = this.tagMappers.reduce((acc, mapper) =>
```

Comment:
The only issue I see with this PR is that for lines that neither match the customParsers nor tagMappers, lines are unnecessarily duplicated. This results in as many data events as there are tagMappers for the same line.

We might only want to concat the result of `mapper(line)` if it is different from the original `line`